core/unit_test/TestViewSubview.hpp

```diff
@@ -2148,6 +2156,22 @@ inline void test_subview_memory_traits_construction() {
   TestSubviewMemoryTraitsConstruction<13>()();
   TestSubviewMemoryTraitsConstruction<14>()();
   TestSubviewMemoryTraitsConstruction<15>()();
+  TestSubviewMemoryTraitsConstruction<16>()();
```

Comment:
Implementing in the operator() instead of the constructor was just dumb (not asking you to change, just venting when I l see the syntax)